CodeEdit/Utils/Limiter.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Limiter.swift
+//  CodeEdit
+//
+//  Created by Austin Condiff on 11/1/24.
+//
+
+import Combine
```

Comment:
I think Khan already wrote some debounce code somewhere? If so can we combine implementations?

Reply:
I didn't find anything in my search.